### DISCUSSION TITLE
Updating Cmder, not just ConEmu and showing version

### DISCUSSION THREAD

**Discussion by Martinspire** [cmderdev/cmder#2875] [Ideas]
So I always like to keep stuff updated and find the ConEmu updating very handy but that doesn't update Cmder itself. I find that very weird. There's also no indication as to what version I'm running and an easy way to just know that there is a new version available.

Are there any plans on adding a page that does this or will Cmder always be quite invisible when you actually have installed the application?

**Comment by DRSDavidSoft** [comment]
I added this to backlog: #2788 

Definitely a good consideration for the overhaul of Cmder

It will be discussed in https://github.com/orgs/cmderdev/discussions/3066

**Comment by daxgames** [comment]
Just to clarify.  Cmder is a product that includes an external product called ConEmu as its terminal emulator.

Cmder is just configuration used ConEmu, so updating ConEmu does NOTHING to update Cmder.

Cmder version x.x.x includes ConEmu version y.y.y.

If you update Cmder, you may or may not get a newer version of ConEmu, but updating ConEmu will never update Cmder.

**Reply by DRSDavidSoft** [reply]
That is correct and OP states the same thing.  To be more precise, it would be useful to be able to:

- find the current version of Cmder configuration without needing to resort to the included Version* files, or checking the Cmder.exe properties for the attached version information

- having the ability to fetch Cmder configuration file updates from remote server without needing to manually download an unpack them. Preferentially this should only overwrite the bundled dependencies if they are found out to be newer than currently existing ones

If we stick with ConEmu, it would be nice if the launcher had the ability to pull Cmder config files from GitHub or something and could also display the information about the Cmder config version using dialogs.

**Reply by daxgames** [reply]
@DRSDavidSoft agree but unfortunately that is almost all beyond my knowledge of C++.

**Reply by DRSDavidSoft** [reply]
@daxgames No problem, I've added this to the backlog.  Once the current open PRs are merged, I'm planning to work on the backlog, and I'll submit PRs for them also, so you can review them and tell me what you think.

Discussions about this feature will held in https://github.com/orgs/cmderdev/discussions/3066
